doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/adding-data-sets-proc.adoc

```diff
@@ -0,0 +1,29 @@
+[id='adding-data-sets-proc_{context}']
+= Adding data sets
+
+You can create a data set to fetch data from an external data source and use that data for the reporting components.
+
+.Procedure
+. In {CENTRAL}, go to *Admin* -> *Data Sets*.
++
+The *Data Set Explorer* page opens.
+. Click *New Data Set* and select one of the following provider types:
++
```

Comment:
Totally up to you, but I prefer the following format for the types for minimalism:

* *Bean:* Generates a data set from a Java class
* *CSV:* Generates a data set from a remote or local CSV file
* *SQL:* Generates a data set from an ANSI-SQL compliant database
* *Elastic Search:* Generates a data set from Elastic Search nodes
* *Prometheus:* Generates a data set using the Prometheus query
* *Execution Server:* Generates a data set using the custom query feature of an Execution Server

Reply:
Excellent point. Thank you :)